apps/web/src/chat/tool-calls.css

```
@@ -31,6 +31,57 @@
+/* Reasoning rendered as a tool-style card (ReasoningCard) — same DS ToolCard chrome,
+   collapsed by default, so the model's native thinking stacks evenly with the tool cards
+   instead of dominating the thread. */
+.reasoning-card {
+  margin: 0 0 0.6em; /* match the tool-call column's bottom margin */
+}
+.reasoning-card .pl-toolcard__status--done {
+  display: none; /* reasoning isn't a pass/fail result — keep only the streaming spinner */
+}
+.reasoning-text {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)